python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -116,11 +117,37 @@ def __init__(self, *args, **kwargs):
         self.as_index = kwargs.pop("as_index", True)
         super().__init__(*args, **kwargs)
 
-    def mean(self, split_every=None, split_out=1):
+    def collect(self, split_every=None, split_out=1):
+        return groupby_agg(
+            self.obj,
+            self.index,
+            {self._slice: "collect"},
+            split_every=split_every,
+            split_out=split_out,
+            dropna=self.dropna,
+            sep=self.sep,
+            sort=self.sort,
+            as_index=self.as_index,
+        )[self._slice]
+
+    def std(self, split_every=None, split_out=1):
         return groupby_agg(
             self.obj,
             self.index,
-            {self._slice: "mean"},
+            {self._slice: "std"},
+            split_every=split_every,
+            split_out=split_out,
+            dropna=self.dropna,
+            sep=self.sep,
+            sort=self.sort,
+            as_index=self.as_index,
+        )[self._slice]
+
+    def var(self, split_every=None, split_out=1):
```

Comment:
I added in overrides for `var` and `std` because it seems like trying to use upstream Dask's implementations here fails:

```
python/dask_cudf/dask_cudf/tests/test_groupby.py ............F
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> traceback >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

aggregation = 'std'

    @pytest.mark.parametrize("aggregation", SUPPORTED_AGGS)
    def test_groupby_basic_series(aggregation):
        pdf = pd.DataFrame(
            {
                "x": np.random.randint(0, 5, size=10000),
                "y": np.random.normal(size=10000),
            }
        )
    
        gdf = cudf.DataFrame.from_pandas(pdf)
    
        ddf = dask_cudf.from_cudf(gdf, npartitions=5)
    
        a = getattr(gdf.groupby("x").x, aggregation)()
>       b = getattr(ddf.groupby("x").x, aggregation)().compute()

python/dask_cudf/dask_cudf/tests/test_groupby.py:61: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../dask/dask/dataframe/groupby.py:1445: in std
    v = self.var(ddof, split_every=split_every, split_out=split_out)
../dask/dask/dataframe/groupby.py:1439: in var
    result = result[self._slice]
../dask/dask/dataframe/core.py:4062: in __getitem__
    meta = self._meta[_extract_meta(key)]
../compose/etc/conda/cuda_11.2/envs/rapids/lib/python3.8/contextlib.py:75: in inner
    return func(*args, **kwds)
python/cudf/cudf/core/dataframe.py:1007: in __getitem__
    return self._get_columns_by_label(arg, downcast=True)
python/cudf/cudf/core/dataframe.py:1863: in _get_columns_by_label
    new_data = super()._get_columns_by_label(labels, downcast)
python/cudf/cudf/core/frame.py:507: in _get_columns_by_label
    return self._data.select_by_label(labels)
python/cudf/cudf/core/column_accessor.py:344: in select_by_label
    return self._select_by_label_grouped(key)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = ColumnAccessor(multiindex=False, level_names=[None])
y: float64, key = 'x'

    def _select_by_label_grouped(self, key: Any) -> ColumnAccessor:
>       result = self._grouped_data[key]
E       KeyError: 'x'

python/cudf/cudf/core/column_accessor.py:406: KeyError
```

Instead of adding these functions in individually, is it possible / would it make sense to add them in programmatically based on `SUPPORTED_AGGS`?